### ISSUE TITLE
How can I add a all privilege user and a read only user?

### ISSUE THREAD

**Issue by xjl456852** [hacdias/webdav#120]
How do I configure it?
I tried several times without success

for example: the readonlyName user can modify,delete, update the /home folder. Cannot control this user as read-only user

```yml
address: 0.0.0.0
port: 8080
auth: true
tls: false
prefix: /
debug: false

scope: .
modify: true
rules: []

users:
  - username: xjl456852
    password: xxx
    scope: /home/
  - username: readonlyName
    password: xxx
    scope: /home/
    modify:   false

```

**Comment by GhostHjh** [comment]
This problem is caused by a code error, and this error is not solved by the author for the time being, but it is solved in this issues

https://github.com/hacdias/webdav/issues/97

**Comment by hacdias** [comment]
The latest commits with the fix (#104) is now released.
